src/main/cli/cli.c

```diff
@@ -189,6 +189,9 @@ static bool configIsInCopy = false;
 #define CURRENT_PROFILE_INDEX -1
 static int8_t pidProfileIndexToUse = CURRENT_PROFILE_INDEX;
 static int8_t rateProfileIndexToUse = CURRENT_PROFILE_INDEX;
+#ifdef USE_OSD_PROFILES
+static int8_t osdProfileIndexToUse = CURRENT_PROFILE_INDEX;
```

Comment:
This is now unused, and should be removed.

Reply:
Oops, yes - thank you.  Now removed.